code/modules/mob/living/simple_animal/abnormality/teth/beanstalk.dm

```diff
@@ -58,13 +58,15 @@
 		step_towards(user, src)
 		sleep(0.5 SECONDS)
 		to_chat(user, "<span class='userdanger'>You start to climb!</span>")
-		//animate(user, alpha = 1,pixel_x = 0, pixel_z = 16, time = 3 SECONDS) works but seemingly can't be undone
+		animate(user, alpha = 1,pixel_x = 0, pixel_z = 16, time = 3 SECONDS)
+		user.pixel_z = 16
```

Comment:
You are instantly setting pixel_z to 16, instead of animating.

Reply:
There's a reason for this, animating doesn't actually move your pixel so what ends up happening is you end up sinking into the ground without instantly setting pixel_z. It "looks" perfectly fine because animate only cares about where your intiial location is.

Reply:
I will, however make the stun a bit longer so that you can't actually start moving your character until they are on the ground.